docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,288 @@
+---
+sidebar_position: 2
+description: 'General Information on EDC Policies'
+title: Working with Policies
+id: connector_kit_adoption_view_policies
+---
+
+## Working with Policies
+
+Policies are a formalization of the terms under which a Provider extends an offer to the Dataspace. The format is given
+by the Dataspace Protocol Specification: The [Open Digital Rights Language (ODRL)](https://www.w3.org/TR/odrl-model/).
```

Comment:
That is not entirely correct. There is the Data Space Protocol Specification:
https://docs.internationaldataspaces.org/dataspace-protocol/
This specification uses some parts of the linked W3C ODRL standard.

The current sentence is misleading since people could think that ODRL is the data space protocol specification which is wrong.

Reply:
Further, Catena-X uses its own format with specializations (like framework agreements). Therefore, a specific policy format from another data space couldn't be understood by a Catena-X connector. I would mention this as a hint or warning.